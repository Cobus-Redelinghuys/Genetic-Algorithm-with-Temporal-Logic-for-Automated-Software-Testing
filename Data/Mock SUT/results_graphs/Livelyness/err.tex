Table \ref{tab:HP:GA:Livelyness:number of errors found} tabulates the final average, standard deviations, and overall average for the number of errors found that the different GATLAST configurations configured with different proper termination weights were able to achieve.
\begin{table}[tbh!]
\centering
\begin{adjustbox}{trim=0cm 0cm 0cm 0cm, clip=true}
\begin{tabular}{|c|c|c|c|}
\hline
proper termination & Final average & Final standard deviation & Overall average\\
\hline
0.0 & 7.1 & 0.568 & 6.838\\\hline
0.25 & 7.2 & 0.422 & 6.98\\\hline
0.5 & 7.2 & 0.422 & 6.756\\\hline
0.75 & 7.0 & 0.0 & 6.968\\\hline
1.0 & 7.1 & 0.316 & 7.0\\\hline
\end{tabular}
\end{adjustbox}
\caption{Final average, standard deviation, and overall average for the number of errors found metric that the of different proper termination configured GATLASTs were achieved}
\label{tab:HP:GA:Livelyness:number of errors found}
\end{table}