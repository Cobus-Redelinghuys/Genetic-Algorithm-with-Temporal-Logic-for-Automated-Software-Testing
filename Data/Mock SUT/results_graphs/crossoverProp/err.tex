Table \ref{tab:HP:GA:crossoverProp:number of errors found} tabulates the final average, standard deviations, and overall average for the number of errors found that the different GATLAST configurations configured with different crossover probability weights were able to achieve.
\begin{table}[tbh!]
\centering
\begin{adjustbox}{trim=0cm 0cm 0cm 0cm, clip=true}
\begin{tabular}{|c|c|c|c|}
\hline
crossover probability & Final average & Final standard deviation & Overall average\\
\hline
0.0 & 7.0 & 0.0 & 6.898\\\hline
0.25 & 7.3 & 0.483 & 6.984\\\hline
0.5 & 7.2 & 0.422 & 6.756\\\hline
0.75 & 7.7 & 0.483 & 7.136\\\hline
1.0 & 7.1 & 0.316 & 6.93\\\hline
\end{tabular}
\end{adjustbox}
\caption{Final average, standard deviation, and overall average for the number of errors found metric that the of different crossover probability configured GATLASTs were achieved}
\label{tab:HP:GA:crossoverProp:number of errors found}
\end{table}