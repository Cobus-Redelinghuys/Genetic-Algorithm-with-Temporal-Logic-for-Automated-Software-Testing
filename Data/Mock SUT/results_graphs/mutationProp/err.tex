Table \ref{tab:HP:GA:mutationProp:number of errors found} tabulates the final average, standard deviations, and overall average for the number of errors found that the different GATLAST configurations configured with different mutation probability weights were able to achieve.
\begin{table}[tbh!]
\centering
\begin{adjustbox}{trim=0cm 0cm 0cm 0cm, clip=true}
\begin{tabular}{|c|c|c|c|}
\hline
mutation probability & Final average & Final standard deviation & Overall average\\
\hline
0.0 & 7.4 & 0.516 & 7.058\\\hline
0.25 & 7.7 & 0.483 & 7.072\\\hline
0.5 & 7.2 & 0.422 & 6.756\\\hline
0.75 & 6.9 & 0.316 & 6.874\\\hline
1.0 & 7.0 & 0.0 & 6.782\\\hline
\end{tabular}
\end{adjustbox}
\caption{Final average, standard deviation, and overall average for the number of errors found metric that the of different mutation probability configured GATLASTs were achieved}
\label{tab:HP:GA:mutationProp:number of errors found}
\end{table}