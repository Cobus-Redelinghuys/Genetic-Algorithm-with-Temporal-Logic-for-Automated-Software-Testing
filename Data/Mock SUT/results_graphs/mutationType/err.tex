Table \ref{tab:HP:GA:mutationType:number of errors found} tabulates the final average, standard deviations, and overall average for the number of errors found that the different GATLAST configurations configured with different mutation type weights were able to achieve.
\begin{table}[tbh!]
\centering
\begin{adjustbox}{trim=0cm 0cm 0cm 0cm, clip=true}
\begin{tabular}{|c|c|c|c|}
\hline
mutation type & Final average & Final standard deviation & Overall average\\
\hline
Bitwise inversion & 7.2 & 0.422 & 6.756\\\hline
Random selection & 7.5 & 0.527 & 7.066\\\hline
Inversion of a single bit & 7.1 & 0.316 & 7.014\\\hline
\end{tabular}
\end{adjustbox}
\caption{Final average, standard deviation, and overall average for the number of errors found metric that the of different mutation type configured GATLASTs were achieved}
\label{tab:HP:GA:mutationType:number of errors found}
\end{table}