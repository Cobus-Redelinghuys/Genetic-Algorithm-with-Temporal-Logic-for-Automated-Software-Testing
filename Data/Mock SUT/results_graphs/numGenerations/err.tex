Table \ref{tab:HP:GA:numGenerations:number of errors found} tabulates the final average, standard deviations, and overall average for the number of errors found that the different GATLAST configurations configured with different number of generations weights were able to achieve.
\begin{table}[tbh!]
\centering
\begin{adjustbox}{trim=0cm 0cm 0cm 0cm, clip=true}
\begin{tabular}{|c|c|c|c|}
\hline
number of generations & Final average & Final standard deviation & Overall average\\
\hline
25 & 6.9 & 0.316 & 6.876\\\hline
50 & 7.2 & 0.422 & 6.756\\\hline
75 & 7.5 & 0.527 & 7.176\\\hline
100 & 7.2 & 0.422 & 7.007\\\hline
150 & 7.5 & 0.527 & 7.243\\\hline
\end{tabular}
\end{adjustbox}
\caption{Final average, standard deviation, and overall average for the number of errors found metric that the of different number of generations configured GATLASTs were achieved}
\label{tab:HP:GA:numGenerations:number of errors found}
\end{table}