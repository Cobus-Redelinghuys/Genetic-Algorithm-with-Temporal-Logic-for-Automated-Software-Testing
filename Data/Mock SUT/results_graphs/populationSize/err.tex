Table \ref{tab:HP:GA:populationSize:number of errors found} tabulates the final average, standard deviations, and overall average for the number of errors found that the different GATLAST configurations configured with different population size weights were able to achieve.
\begin{table}[tbh!]
\centering
\begin{adjustbox}{trim=0cm 0cm 0cm 0cm, clip=true}
\begin{tabular}{|c|c|c|c|}
\hline
population size & Final average & Final standard deviation & Overall average\\
\hline
25 & 7.0 & 0.471 & 6.93\\\hline
50 & 7.2 & 0.422 & 6.756\\\hline
75 & 7.3 & 0.675 & 6.86\\\hline
100 & 6.6 & 0.516 & 6.5\\\hline
125 & 7.1 & 0.316 & 6.6\\\hline
150 & 6.7 & 0.483 & 6.554\\\hline
\end{tabular}
\end{adjustbox}
\caption{Final average, standard deviation, and overall average for the number of errors found metric that the of different population size configured GATLASTs were achieved}
\label{tab:HP:GA:populationSize:number of errors found}
\end{table}