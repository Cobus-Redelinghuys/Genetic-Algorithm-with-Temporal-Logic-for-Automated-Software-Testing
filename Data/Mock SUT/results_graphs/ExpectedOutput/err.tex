Table \ref{tab:HP:GA:ExpectedOutput:number of errors found} tabulates the final average, standard deviations, and overall average for the number of errors found that the different GATLAST configurations configured with different expected output weights were able to achieve.
\begin{table}[tbh!]
\centering
\begin{adjustbox}{trim=0cm 0cm 0cm 0cm, clip=true}
\begin{tabular}{|c|c|c|c|}
\hline
expected output & Final average & Final standard deviation & Overall average\\
\hline
0.0 & 7.2 & 0.422 & 6.786\\\hline
0.25 & 7.2 & 0.422 & 7.018\\\hline
0.5 & 7.2 & 0.422 & 6.756\\\hline
0.75 & 7.0 & 0.0 & 6.988\\\hline
1.0 & 7.1 & 0.316 & 6.998\\\hline
\end{tabular}
\end{adjustbox}
\caption{Final average, standard deviation, and overall average for the number of errors found metric that the of different expected output configured GATLASTs were achieved}
\label{tab:HP:GA:ExpectedOutput:number of errors found}
\end{table}